src/components/main/Main.tsx

```diff
@@ -0,0 +1,17 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from './TodoItem';
+
+interface Props {
+  todos: Todo[];
+}
+export const Main: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+      {todos ? todos.map(todo => (
```

Comment:
you can set todo property as empty array by default and no worries about additional checking
```suggestion
      {todos.map(todo => (
```

Reply:
your previous suggestions fixed it as well